### DISCUSSION TITLE
Seems like the example on `nickel-lang.org` is broken?

### DISCUSSION THREAD

**Discussion by edude03** [tweag/nickel#557] [Q&A]
I wanted to try out the nickel lang demo, but it seems like unintentionally, strings.isMatch doesn't work. I tried to see if `strings.isMatch` had changed but I also couldn't find docs for the standard lib either 😆

**Comment by Avi-D-coder** [comment]
We recently switched to snake_case try `strings.is_match`.

**Comment by yannham** [comment]
Thanks for reporting. Indeed, the example is broken and we should fix it. In the meantime, to see the functions available in the strings stdlib, you can use `nickel query strings` in the command line or `:query strings` in the REPL. The latter also works inside the online playground (when selecting the REPL tab, and not JSON or YAML. You can also see the specific documentation about a stdlib function using `:query strings.is_match` for example.
